gatsby/src/components/Accordion/Accordion.module.scss

```diff
@@ -58,14 +58,24 @@
   summary:hover & {
     color: core.$nds-colour-link-hover;
   }
+
+  .warningTheme & {
+    color: core.$nds-colour-nice-rich-black;
+  }
 }
 
 .icon {
   display: block;
   margin-right: core.rem(core.$nds-spacing-small);
   transition: all 0.25s ease;
 
-  [open] & {
+  /* stylelint-disable-next-line selector-max-compound-selectors */
+  [open] > summary > .toggleLabel & {
     transform: rotate(-180deg);
   }
 }
+
+.warningTheme {
+  background: #fef5e5; // TODO: Add to design system
```

Comment:
Can we use the new colours from the new NDS release and remove the TODOs please?